src/main/java/org/forgerock/openam/auth/nodes/EmailNotifyNode.java

```diff
@@ -99,18 +102,18 @@ public Action process(TreeContext context) throws NodeProcessException {
         // Override email "to" field if found in sharedState
         if (context.sharedState.get("email").asString() != null) {
             emailAddr = context.sharedState.get("email").asString();
-            debug.error("[" + DEBUG_FILE + "]: " + "got email address from sharedState: " + emailAddr);
+            debug.error("[" + NODE_NAME + "]: " + "got email address from sharedState: " + emailAddr);
```

Comment:
Marketplace nodes should not log debug stuff on error level, that's just really not acceptable for prod environments.